config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,21 @@
+redis_config = {
```

Comment:
Thanks @tijmenb I can see this initializer is a duplicate of same in govuk_sidekiq, will remove.

As regards to your point about the dependency between content store and redis, I agree, it feels like the tests in the async_experiments gem needs to cover the scenario of redis being unavaible and handling this gracefully. I'll put this PR on hold until I've got those changes in.